frontend/src/hooks/useToken.js

```diff
@@ -9,12 +9,18 @@ export default function useToken() {
   const [token, setToken] = useState(getToken());
 
   function saveToken(token) {
+    console.log("savetoken")
+    if (token === null) {
+      sessionStorage.clear('acces_token');
+      setToken("")
+      return;
+    }
     sessionStorage.setItem('token', token);
     setToken(token);
   };
 
   return {
     setToken: saveToken,
-    token
+    token,
```

Comment:
a végére szerintem nem kell a vessző, de ha van máshogy beállított formatter/linter akkor úgy a helyes

Reply:
Ezt Roland mondta nekünk, hogy legyen így, mert akkor ha újabb dolgot rakunk a return-be, akkor ez a sor nem változik.

Reply:
ez marad így :) megegyezés alapján így használjuk